sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicLogicalOperators.scala

```diff
@@ -817,6 +818,18 @@ case class Sample(
     child: LogicalPlan)(
     val isTableSample: java.lang.Boolean = false) extends UnaryNode {
 
+  val eps = RandomSampler.roundingEpsilon
```

Comment:
just follow the way RandomSampler.scala handles floating numbers: "epsilon slop to avoid failure from floating point jitter"